### PR TITLE
[libfix] Add ceph_instance to run.py

### PR DESCRIPTION
**Pull request by adityaramteke**
Add RHOS-d instance name to run.py. 

It'll help in accessing the instance name through test case

### CODE REVIEW

File: cli/cloudproviders/openstack.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def get_nodes(self, refresh=False):
         if refresh or not self._nodes:
```
Comment (vamahaja) [inline cli/cloudproviders/openstack.py:111]: These changes are not required now. Please revert.

File: cli/cluster/node.py [hunk 2]
```diff
@@ -17,17 +17,7 @@ def __init__(self, name, cloud):
         **kwargs:
```
Comment (vamahaja) [inline cli/cluster/node.py:17]: These changes are not required now. Please revert.

File: ceph/ceph.py [hunk 3]
```diff
@@ -47,6 +47,7 @@ def __init__(self, name, node_list=None):
         self.custom_config = None
```
Comment (vamahaja) [inline ceph/ceph.py:47]: why this change required ?

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: cli/cluster/node.py (hunk 2 above)
Comment (adityaramteke) [reply cli/cluster/node.py:17]: Done, reverted the changed.

**Review by tintumathew10 — COMMENTED** [review]
LGTM

File: run.py [hunk 4]
```diff
@@ -237,6 +237,7 @@ def create_nodes(
                     private_ip=private_ip,
                     hostname=node.hostname,
                     ceph_vmnode=node,
+                    ceph_nodename=node.node.name,
```
Comment (vamahaja) [inline run.py:240]: From where this `node.node.name` coming from ? I see `node-name` variable here - https://github.com/red-hat-storage/cephci/blob/ebfe67ee0fab06bdd242662df4d5466f22e410dc/ceph/utils.py#L340

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: run.py (hunk 4 above)
Comment (adityaramteke) [reply run.py:240]: Not sure, how it is! But from the dir it was coming like that. Here are the logs: http://magna002.ceph.redhat.com/cephci-jenkins/aramteke/automation-83575600/cephci-run-RQD2U9/
